src/gui/DatabaseSettingsWidget.cpp

```diff
@@ -149,3 +166,24 @@ void DatabaseSettingsWidget::truncateHistories()
         entry->truncateHistory();
     }
 }
+
+bool DatabaseSettingsWidget::hasBeenModified() const
+{
+    const int historyMaxItems = m_ui->historyMaxItemsCheckBox->isChecked() ? m_ui->historyMaxItemsSpinBox->value()
+                                                                           : -1;
```

Comment:
Formatting: don't make a new line for 6 characters. Either split earlier or don't split at all.